sbtplugin/src/main/scala/com/typesafe/tools/mima/plugin/MimaPlugin.scala

```diff
@@ -40,14 +40,25 @@ object MimaPlugin extends AutoPlugin {
       }
     },
     mimaReportBinaryIssues := {
-      SbtMima.reportErrors(
-        mimaFindBinaryIssues.value,
-        mimaFailOnProblem.value,
-        mimaBinaryIssueFilters.value,
-        mimaBackwardIssueFilters.value,
-        mimaForwardIssueFilters.value,
-        streams.value,
-        name.value)
+      mimaPreviousClassfiles.value.foreach {
```

Comment:
If you're not going to run `mimaFindBinaryIssues`, don't we probably want to still return the "previous-artifact not set, not analyzing binary compatibility" info message?